### PR TITLE
Css cleanup

### PR DESCRIPTION
**Pull request by tinabeans**

### CODE REVIEW

File: sass/give.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import 'colors';
+@import 'typography';
```
Comment (awong-dev) [inline sass/give.scss:2]: At some point should give.scss share more of style.scss do you think? Not asking a change in this PR. Just trying to understand direction.

File: sass/colors.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+$text-color: #333333;
```
Comment (nburt) [inline sass/colors.scss:1]: 🎉

File: sass/give.scss (hunk 1 above)
Comment (nburt) [reply sass/give.scss:2]: Let's aim to do that as part of the v2 embed work: https://github.com/findthemasks/findthemasks/issues/649

File: sass/style.scss [hunk 3]
```diff
@@ -226,13 +228,13 @@ h3.city {
...
+  padding: 0;
+  margin: 0;
 }
 
 .locations-list-header {
-  padding: 0 !important;
-  margin: 0 !important;
+  padding: 0;
+  margin: 0;
   border-bottom: 1px solid #D4D4D4;
```
Comment (nburt) [inline sass/style.scss:238]: Can you make this a variable too?

File: sass/style.scss [hunk 4]
```diff
@@ -314,19 +317,26 @@ h3.city {
   border-top: 1px solid darkgray;
```
Comment (nburt) [inline sass/style.scss:317]: this gray should mark the border above

**Comment by nburt** [comment]
Added another color var and merged in master

**Pull request merged**